source/Octo/Commands/ZipPackageBuilder.cs

```diff
@@ -47,7 +47,11 @@ public void BuildPackage(string basePath, IList<string> includes, ManifestMetada
 
                         var relativePath = UseCrossPlatformDirectorySeparator(
                             fullFilePath.Substring(basePathLength).TrimStart(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar));
-                        log.Debug("Adding file: {Path}", relativePath);
+
+                        if (verboseInfo)
+                            log.Information("Adding file: {Path}", relativePath);
```

Comment:
Does Log have a `Write(LogLevel, string, params string[])` override you can use instead? (The message is exactly the same)